Formula/hof.rb

```diff
@@ -0,0 +1,46 @@
+class Hof < Formula
+  desc "Flexible data modeling & code generation system"
+  homepage "https://hofstadter.io/"
+  url "https://github.com/hofstadter-io/hof.git",
+    tag:      "v0.6.6",
+    revision: "0f49b4d71a66788b006daacf905a2b138768beca"
```

Comment:
```suggestion
  url "https://github.com/hofstadter-io/hof.git",
      tag:      "v0.6.6",
      revision: "0f49b4d71a66788b006daacf905a2b138768beca"
```
Small nit: align the arguments to `url`.

Reply:
@carlocab Do you want the commits squashed so there is only one ever for this PR?

Reply:
Squashing commits would be good. But squashing here is simple, so @BrewTestBot can also handle that. (Sometimes it's not simple, and needs to be done manually.)

Reply:
I pushed a second commit, hoping that the bots can handle this case 🤞